en/flight_modes/README.md

```diff
@@ -356,6 +356,19 @@ th {
 </tr>
 
 
+<tr id="orbit_mc">
+ <td><a href="../flight_modes/orbit.md">Orbit</a>
+ <p><a href="#key_difficulty"><img src="../../assets/site/difficulty_easy.png" title="Easy to fly" width="20px" /></a></p>
+ </td>
+ <td>-</td>
+ <td>-</td>
+ <td>-</td>
+ <td></td>
+ <td><p>GCS/RC mode for flying a circle, always facing the centre.</p> 

```

Comment:
No offense, I really like British English. I'm only suggesting to change it to match the code and in the code it's Ami because I thought that's more generally understandable to say it carefully.
```suggestion
 <td><p>GCS/RC mode for flying a circle, always facing the center.</p> 
```